packaging/installer/methods/manual.md

```diff
@@ -206,6 +206,10 @@ You can edit this file to set options. One common option to tweak is `history`,
 
 To apply the changes you made, you have to restart Netdata.
 
+### 'nonrepresentable section on output' errors
+
+Our current build process unfortunately has some issues when using certain configurations of the `clang` C compiler on Linux. If the installation fails with errors like `/bin/ld: externaldeps/libwebsockets/libwebsockets.a(context.c.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC` and you are trying to build with `clang` on Linux, you will need to instead build using GCC to get a fully functional install of Netdata. In most cases, you can do this by running `CC=gcc ./netdata-installer.sh`.
```

Comment:
I don't have any objections to placing this workaround here vs. anywhere else. This suggestion is mostly formatting—there's a backtick in the example output that blows up the inline code blocks, so I just changed that to a straight apostrophe. I also broke this up into some paragraphs to help distinguish between problem and solution.

```suggestion
Our current build process unfortunately has some issues when using certain configurations of the `clang` C compiler on Linux.

If the installation fails with errors like `/bin/ld: externaldeps/libwebsockets/libwebsockets.a(context.c.o): relocation R_X86_64_32 against '.rodata.str1.1' can not be used when making a PIE object; recompile with -fPIC`, and you are trying to build with `clang` on Linux, you will need to build Netdata using GCC to get a fully functional install. 

In most cases, you can do this by running `CC=gcc ./netdata-installer.sh`.
```